### PR TITLE
x11-wm/openbox: use 'python-single-r1_pkg_setup' and sync 9999

### PR DESCRIPTION
**Pull request by Chiitoo**
There doesn't seem to be a https source for the homepage or git, but there is a GitHub mirror, so I wonder if that should be used for the 9999...

### CODE REVIEW

File: x11-wm/openbox/openbox-9999.ebuild [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
-inherit multilib autotools ltprune python-r1 eutils
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit multilib autotools python-single-r1 eutils
 
 DESCRIPTION="A standards compliant, fast, light-weight, extensible window manager"
 HOMEPAGE="http://openbox.org/"
```
Comment (a17r) [inline x11-wm/openbox/openbox-9999.ebuild]: redirects to `http://openbox.org/wiki/Main_Page`, might as well use that then.

File: x11-wm/openbox/openbox-9999.ebuild [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
-inherit multilib autotools ltprune python-r1 eutils
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit multilib autotools python-single-r1 eutils
```
Comment (a17r) [inline x11-wm/openbox/openbox-9999.ebuild]: it seems both multilib and eutils would be superfluous

File: x11-wm/openbox/openbox-9999.ebuild [hunk 3]
```diff
@@ -16,7 +16,7 @@ if [[ ${PV} == *9999* ]]; then
 else
 	SRC_URI="http://openbox.org/dist/openbox/${P}.tar.gz
 	branding? ( https://dev.gentoo.org/~hwoarang/distfiles/surreal-gentoo.tar.gz )"
```
Comment (a17r) [inline x11-wm/openbox/openbox-9999.ebuild]: duplicate URI, could be moved out of the whole block via `SRC_URI+=...`

File: x11-wm/openbox/openbox-9999.ebuild [hunk 4]
```diff
@@ -41,17 +41,24 @@ RDEPEND="dev-libs/glib:2
...
-		dev-python/pyxdg[${PYTHON_USEDEP}]
+		$(python_gen_cond_dep '
+			dev-python/pyxdg[${PYTHON_MULTI_USEDEP}]
+		')
 	)
 	"
 DEPEND="${RDEPEND}
 	sys-devel/gettext
 	virtual/pkgconfig
 	x11-base/xorg-proto"
```
Comment (a17r) [inline x11-wm/openbox/openbox-9999.ebuild]: good opportunity to make closing quotes style consistent, personally I prefer newline + 0 indent

**Comment by Chiitoo** [comment]
Thanks!

All done, with additionally adding a line change after HOMEPAGE, splitting keywords due to line length (does it look too weird like that?), and a src_unpack to fix installing (or unpacking rather) the 'Surreal Gentoo' theme for 9999.

**Comment by Chiitoo** [comment]
Oh, and dropping '3.6.1-r1' is new, too.

**Comment by Chiitoo** [comment]
Now with less splitting of lines.

**Pull request closed without merging**